### PR TITLE
Specify back to app feature

### PR DESCRIPTION
**Pull request by IniZio**
ref: #2768

### CODE REVIEW

File: docs/specs/default-redirection.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Fallback redirection scenerios
+
+### Magic Link verification result
+
+When an application initiates Magic Link authentication, it sends a link that initiates the passwordless verification process.
+
+This process can happen in another device / session, but only the original session will be granted access. The other device will reach "dead-end" screen, by default end-user will be prompted to close the window.
+
+When `fallbackRedirectURI` is provided in `authgear.configure`, a "Back to App" button will be displayed at the original "dead-end" screen for returning to client app.
```
[the remark this answers is not in the document]
Comment (louischan-oursky) [reply docs/specs/default-redirection.md:21]: Another note. Very long ago, our verification flow does not send a OTP, but instead send an verification link. That link when clicked leads to a dead-end as well. We worked around the issue by sending a OTP, thus keep the end-user stay in the app. So this feature could make bringing back the verification link behavior.

[the remark this answers is not in the document]
Comment (IniZio) [reply docs/specs/default-redirection.md]: Sure updated the spec

[the remark this answers is not in the document]
Comment (carmenlau) [reply docs/specs/default-redirection.md]: > Or one universal link per dead end?

In the updated spec, I found we have 1 URI only. But I think `one universal link per dead end` makes sense? Imagine in the scenarios, the developer needs to implement a web page in the universal link for the user who hasn't installed the app. They may want to display different content per dead end.

[the remark this answers is not in the document]
Comment (IniZio) [reply docs/specs/default-redirection.md]: I think if there are screens outside of auth flow that want to display this anchor, having `one universal link per dead end` will surprise user which is probably not desirable?

**Comment by IniZio** [comment]
After discussion, will close this PR but keep the branch first.

**Pull request closed without merging**